server/pulp/server/managers/status.py

```diff
@@ -51,9 +54,14 @@ def get_broker_conn_status():
     # explicit connection and then release it.
     # See https://github.com/celery/kombu/issues/432 for more detail.
     try:
-        conn = celery.connection()
         conn.connect()
+        transport = conn._transport
         conn.release()
+
+        # An explicit call to __del__ to release file descriptors
+        # https://pulp.plan.io/issues/2124#note-9
+        transport.__del__()
```

Comment:
Does it make sense to throw this in a `finally:` block, or is there really no way ever that the connection will fail and we'll have file descriptors to clean up?